src/components/TabList/TabsList.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+import { TabsLink } from '../TabsLink';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabs?: string;
```

Comment:
pay attention to comments, rename is as `selectedTabId`, it's only one tab id, not an array of tabs